internal/cmd/testdata/scripts/templatedata.txtar

```diff
@@ -4,7 +4,7 @@
 
 # test that .chezmoi.sourceFile is set
 exec chezmoi cat $HOME${/}.file
-stdout dot_file.tmpl
+cmp stdout golden/dot_file
```

Comment:
This line needs to be

    cmpenv stdout golden/dot_file

Reply:
Whoops. I thought I had copied things correctly.